app/server/api/dtpr/v0/[locale].ts

```diff
@@ -6,7 +6,7 @@ export default defineEventHandler(async event => {
   const { siteUrl } = useRuntimeConfig();
   const locale = event.context.params?.locale || 'en';
 
-  const iconUrl = (icon: string) => {
+  const iconUrl = (icon: /dtpr-icons/string) => {
```

Comment:
This shouldn't change. I think a search and replace got here by accident.

Reply:
Oh, thanks for catching this you're completely right